packages/atlas/src/components/_crt/RevenueShareStateWidget/RevenueShareStateWidget.tsx

```diff
@@ -1,11 +1,14 @@
 import { FlexBox } from '@/components/FlexBox'
 import { Text } from '@/components/Text'
 import { WidgetTile } from '@/components/WidgetTile'
+import { useBlockTimeEstimation } from '@/hooks/useBlockTimeEstimation'
 import { formatDateTime, formatDurationShort } from '@/utils/time'
 
-export const RevenueShareStateWidget = ({ end }: { end?: Date }) => {
-  const status: 'active' | 'past' | 'inactive' = !end ? 'inactive' : end.getTime() < Date.now() ? 'past' : 'active'
-
+export const RevenueShareStateWidget = ({ endsAtBlock }: { endsAtBlock?: number }) => {
+  const { convertBlockToMsTimestamp } = useBlockTimeEstimation()
+  const status: 'active' | 'past' | 'inactive' = !endsAtBlock ? 'inactive' : endsAtBlock < 0 ? 'past' : 'active'
```

Comment:
according to the name of the var `endsAtBlock` it would be the name of the block and the logic for showing the past status should be `endsAtBlock - currentBlock < 0`?